python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1209,6 +1210,237 @@ def bitwise_not(x, out=None, **kwargs):
     return _unary_func_helper(x, _npi.bitwise_not, _np.bitwise_not, out=out, **kwargs)
 
 
+@set_module('mxnet.symbol.numpy')
+def copyto(dst, src, casting='same_kind', where=True):  # pylint: disable=W0621
+    r"""
+    Copies values from one array to another, broadcasting as necessary.
+
+    Raises a TypeError if the `casting` rule is violated, and if
+    `where` is provided, it selects which elements to copy.
+
+    Parameters
+    ----------
+    dst : ndarray
+        The array into which values are copied.
```

Comment:
You're modifying `dst` inplace. How is this expressed in the nnvm graph?

Reply:
My guess is that it'll be similar to how the `out` argument is handled in all previous nd operators?